### DISCUSSION TITLE
create-notion-blog

### DISCUSSION THREAD

**Discussion by giscus** [daeho-ro/notion-blog#4] [Comments]
# create-notion-blog

Notion에 글을 관리하고 Next.js로 Vercel이나 Github Pages에 배포하여 무료로 블로그를 구축할 수 있음을 알게 되었다.

https://blog.lamanus.kr/create-notion-blog

**Comment by Yejin715** [comment]
안녕하세요.
올려주신 글을 참고하여 블로그를 생성 중에 있습니다.
작업 도중 몇 가지 궁금한 점이 생겨 이렇게 댓글로 문의드립니다.

1. 도메인 사용
  - 제가 이해하기로는 도메인을 별도로 유료로 구매하지 않는 경우, 기본적으로 "vercel.app" 도메인을 사용해야 하는 것 같습니다. 혹시 맞는지 확인 부탁드립니다.

2. 구글 검색 가능하도록 제작
  - 작성해주신 내용과 사용하신 라이브러리를 기반으로 블로그를 제작했지만, 현재 제 블로그가 구글 검색에 노출되지 않는 것으로 보입니다.
  - 검색이 가능하도록 하려면 "검색엔진 등록"이 필요하다는 정보를 확인했는데, 이 과정에서 메타 태그를 추가해야 한다고 들었습니다.
혹시 작성자님께서는 이 방법으로 진행하셨는지, 아니면 다른 방법을 사용하셨는지 궁금합니다.

제가 블로그 제작에 처음 도전하다 보니 기본적인 질문을 드리게 된 점 양해 부탁드립니다.
답변 주시면 큰 도움이 될 것 같습니다. 감사합니다!

**Reply by daeho-ro** [reply]
따로 알림을 설정해두지 않아서 이제서야 댓글을 확인하게 되었습니다.

1. 도메인은 별도 구매가 맞습니다. 저는 오래전부터 사용하는 도메인이 있어서 그대로 사용했는데, 별도의 도메인이 없으면 기본 제공되는 `vercel.app` 도메인을 사용해야 하는 것이 맞습니다.

2. 메타태그 등록은 템플릿을 직접 수정하면 되기는 합니다. `PageHead.tsx` 파일에서 편집하면 될 것 같습니다. 저는 별도 도메인이 있기 때문에 도메인 인증 방식으로 처리되었습니다.
```
> dig lamanus.kr txt +short
"google-site-verification=UllLAZNPpjvWyFwXOnzF50LjdhORZVRzr78jC03bw7c"
"v=spf1 -all"
```

**Reply by Yejin715** [reply]
아하 감사합니다!!
확인해보고 한번 시도해보도록 하겠습니다!
2024년도도 고생 많으셨습니다🫡
